### ISSUE TITLE
Single-click to edit text (instead of double-click)

### ISSUE THREAD

**Issue by sunnyt7** [soflyy/oxygen-bugs-and-features#205]
I think Oxygen is the only builder I've used that requires you to double-click to edit text.

For some reason, I feel like this breaks the flow up a bit - especially when going from one text box to another. I also can't think of any downsides of going to single click?

Almost 80% of people in the FB group voted that they prefer single click over double-click. This would also be paired well with https://github.com/soflyy/oxygen-bugs-and-features/issues/203.

**Comment by timpixel** [comment]
Totally agree, but how would you edit the text in that case? In a separate field in the options panel?

**Comment by sunnyt7** [comment]
> 
> 
> Totally agree, but how would you edit the text in that case? In a separate field in the options panel?

Once you click on the text, it would initiate the text cursor. Meaning you could start typing, highlight etc. That's how most other page builders function.

**Comment by timpixel** [comment]
Ok, is the text element still draggable then? Perhaps the "click on the text" works on mouse-release, and drag works on mouse-down and drag... Anyway I'm sure that will be fine.

**Comment by sunnyt7** [comment]
> 
> 
> Ok, is the text element still draggable then? Perhaps the "click on the text" works on mouse-release, and drag works on mouse-down and drag... Anyway I'm sure that will be fine.

Ah, I don't use the dragging much (since it's pretty buggy and there's no undo) so I hadn't thought of that.

Sounds like a good solution though. Or have a corner of the textbox that's draggable - indicated by an icon on hover.

**Comment by timpixel** [comment]
Good idea, most elements have a small label as well, that shows the name. Perhaps that could be a drag handle also. But I rarely use drag as well, like you. Best thing I believe would be if the structure panel can always be used (bug-free), instead of dragging.
